user/reference/overview.md

```diff
@@ -257,9 +257,10 @@ Each tier of instance size delivers more vCPU and RAM resources available at you
 
 | size      | vCPU        | Memory GiB  | Comment                                                 |
 |:---------:|:-----------:|:-----------:|:-------------------------------------------------------:|
+| medium    | 2           | ~8          | default                                                 |
```

Comment:
if you're adding `medium` here as default, please extend comment: from `default` to `default and assumed automatically if no size: medium present in the .travis.yml` or something along those lines